src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -43,6 +43,9 @@
 from promptflow.executor.flow_executor import DEFAULT_CONCURRENCY_BULK, FlowExecutor
 from promptflow.storage import AbstractRunStorage
 
+LogName = "process_stderr"
```

Comment:
In python, the naming convention of constants is capital letters with underscore separators. I.e. "LOG_NAME"
https://peps.python.org/pep-0008/#constants

Reply:
fix